src/Bicep.Core.Samples/Files/InvalidTypeDeclarations_LF/main.sourcemap.json

```diff
@@ -1,4 +0,0 @@
-{
-  "Entrypoint": "main.bicep",
-  "Entries": []
-}
```

Comment:
Not related to this PR: The JSON files for this data set are probably just a survival from an early version of the initial types PR. I'll try removing `main.json` and `main.symbolicnames.json` and make sure they aren't regenerated by the baseline script.